### ISSUE TITLE
PC Frozen Chambers Difficulty?

### ISSUE THREAD

**Issue by FollowTheTrain-CJ** [Garmundo/Extended-unit-arsenal#11]
After starting the map,when i started mining CL.i noticed that something was wrong, [phantoms] and revelants swarmed my base and the salvos didnt stand a chance,the maestorm got its ammo supplied cut and they can hardly hit phantoms,i think there is some balance issue on this map,otherwise it will be a masochism scaled difficulty map
https://prntscr.com/tvhnaq

**Comment by Garmundo** [comment]
Was this after the fix? Check if there's a scrap deposit in the starting area, If there is, it should be the fixed map.

**Comment by FollowTheTrain-CJ** [comment]
Yep,i got the newest version from kind-heart guys in discord,i kept downloading the zip from github big green button and i got 5 zip files that are all old versions,in the new version there wasnt such a horrible swarm,thanks

**Comment by Garmundo** [comment]
So the map in the github download is still the one with the swarm? I downloaded recently and I didn't see the swarm come up. I'm getting really confused.

**Comment by FollowTheTrain-CJ** [comment]
https://prnt.sc/tvrgg2
the 2 on topthe versions i got from github earlier today
the different one is from discord players
i update the game by replacing the zip files in the /mods and it is working for other mods,do u use the scrap deposit to identify the new version?check on the minimap in one of my uploaded images,i think u can see the scrap stuff there

**Comment by Garmundo** [comment]
I added in the scrap deposit the same time I fixed the conveyors which trigger the factories so the swarm should be resolved. There should be 2 item voids next to the phase-fabric item source leading to the 6 phantasm factories in the section between the thorium area and the surge area. I wonder if its because the mechanics are different from mobile. Do you have a mod that changes conveyor belt behavior? If not there must be some kind of difference between mobile and pc. The other factories at the surge also should have item voids next to the item sources, if there is a constant flow of nebulas and revenants maybe the voids didnt show up for some reason.

**Comment by FollowTheTrain-CJ** [comment]
vanilla addon mod does have an extra conveyor called hermetic belt,but it didnt has nothing to do to titanium belts,i will check the void and belt for you,see if they are updated,i feel like i am having problem updating the zip file

**Comment by FollowTheTrain-CJ** [comment]
yes,some discord guy told me that the map was only playable in mobile,that might be the issue,i am sure there is a scrap deposit in my starting area

**Comment by FollowTheTrain-CJ** [comment]
https://prntscr.com/tw4pcl
Scrap deposit,is this the one you are talking about?
https://prntscr.com/tw4p8w
This shot shows that the enemy starts producing LICH at the very start of the game

**Comment by Garmundo** [comment]
At this point I have no clue how this is happening. The layout in the second pic looks exactly like the way I set it up but the belts have clogged for some reason. That shouldn't be happening. 

But you said the one you got from discord players work fine? Because when I play it it works alright. I am so confused : (

**Comment by FollowTheTrain-CJ** [comment]
well,their version does not work as well,the swarm does happen,l think its the platform's problem,in my case pc,i guess i wont be playing this map until the problem is fixed,too shame.I found it a very potential map full of resources that i can farm enough resource for new technology...
However other maps are full of joy to play with,keep up!!

**Comment by Garmundo** [comment]
I guess you could just avoid this map for now. I can't balance it properly without a pc to see the problem firsthand. Im glad if you enjoy the rest of the mod though
